### PR TITLE
[ST-388]  Adding crop's name on "Actual Revenue" 

### PR DESCRIPTION
**Pull request by hernandcb**
Added the name of the crop as complement to the Crop variety name in the Actual revenue page.
I used the same pattern used in the estimated crop revenue.

### CODE REVIEW

File: packages/webapp/src/containers/Finances/ActualCropRevenue/index.js [hunk 1]
```diff
@@ -35,6 +41,11 @@ const ActualCropRevenue = ({ sale, history, ...props }) => {
         const convertedQuantity = roundToTwoDecimal(
           convertFromMetric(cvs.quantity.toString(), cvs.quantity_unit, 'kg').toString(),
         );
+        const cropVariety =  cropVarietyEntities[cvs.crop_variety_id];
+        const title = cropVariety.crop_variety_name
```
Comment (smattingly) [inline packages/webapp/src/containers/Finances/ActualCropRevenue/index.js:45]: The value of `title` is never used. You need to use it in the returned object.

**Review by smattingly — CHANGES_REQUESTED** [review]
You've put together the proper string, but haven't made it display.

**Comment by hernandcb** [comment]
Oh, I don't know why I sent it like that. Fixed it, also added the key prop to avoid a React warning that was appearing.

**Comment by JimmyRowland** [comment]
Couldn't push onto this branch. Moved this pr here https://github.com/LiteFarmOrg/LiteFarm/pull/1733

**Pull request closed without merging**